src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -255,6 +255,27 @@ describe('Test suite for dictionary concepts components', () => {
     expect(mapStateToProps(initialState).newName).toEqual(['1']);
     expect(mapStateToProps(initialState).existingConcept).toEqual(existingConcept);
   });
+
+  it('should show the modal when cancel is clicked with unsaved changes', () => {
+    const wrapper = mount(<Router><EditConcept {...props} /></Router>);
+    const event = {
+      target: {
+        name: 'datatype',
+        value: 'Complex',
+      },
+    };
+    wrapper.find('select#datatype').simulate('change', event);
+    wrapper.find('button#remove').simulate('click');
+    expect(wrapper.find('EditConcept').instance().state.show).toEqual(true);
```

Comment:
Why are you repeating 'EditConcept' in several places?

Reply:
I have created a variable to avoid this repetition